CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 0.3.4 - 2018/5/10
+- Only include slug from the soundcloud URL
```

Comment:
Might be good to flag this as a **Fix** release and also mention that this is related to the Google Doc parsing:
```
- **Fix:** When parsing a Google Document, only store slug from Soundcloud URL
```

Reply:
+!

Reply:
+1

Reply:
updated

Reply:
It would be great if you could also include the _Google Document parsing_ part (see example above) and maybe squash your two commits to keep the repository history clean (to me, the `git log` is even more important here as it's an open source repository).